src/Exception/ParseException.php

```diff
@@ -10,10 +10,10 @@ final class ParseException extends SchemaException
     private const CLASS_ERROR = 'Class "%s" couldn\'t be parsed because of an error: %s';
     private const PROPERTY_ERROR = 'Property "%s::%s" couldn\'t be parsed because of an error: %s';
     private const PROPERTY_TYPE_ERROR = 'Property "%s::%s" has an invalid type which results in: %s';
-    private const PROPERTY_TYPE_NAME_NULL = 'Property "%s::%s" has an invalid type annotation. [Type Error] Attribute "name" of @JMS\Type may not be null.';
+    private const PROPERTY_TYPE_NAME_NULL = 'Property "%s::%s" has an invalid type attribute. [Type Error] Attribute "name" of @JMS\Type may not be null.';
```

Comment:
```suggestion
    private const PROPERTY_TYPE_NAME_NULL = 'Property "%s::%s" has an invalid type attribute. [Type Error] Attribute "name" of #JMS\Type may not be null.';
```
should we change the @ to # to match that we talk about attribute and not annotation?

Reply:
I agree and I adjusted it accordingly.